ops/ccs-ops-misc/load_test/README.md

```diff
@@ -110,7 +110,37 @@ Essentially, all the items you would set up in the config file are set in a sing
 
+<<<<<<< HEAD
 **--resetStats** : (Optional) : If this flag is included, the test statistics will reset to zero after clients have finished spawning. **Note:** There are many reasons why we might want to capture statistics while new load is being added. There might be performance problems accepting the connection or new connections might affect users already connected to the system.
+=======
+### Data setup for contract tests
+
+There is one special type of test that requires a data setup script to be run beforehand; this is the coverage contract test. This test runs through every list of pages for a set of contracts, but the urls for hitting all the pages of whatever contracts need to be set up before the test runs. This can be done by calling the data setup script common/write_contract_cursors.py similar to this:
+
+> **Note:** You _must_ generate the contract URLs via the below script if you are running any of the following test suites:
+> * `v1/patient_test_coverageContract.py`
+> * `v1/regression_suite.py`
+> * `v2/patient_test_coverageContract.py`
+> * `v2/regression_suite.py`
+
+    python3 ./common/write_contract_cursors.py --contracts="Z9997,Z9998,Z9999" --year="2020" --month="01" --count="5" --version="v2"
+
+The script will write out the results to homePath specified in the config (and expect them there during runtime.)
+
+The following are the arguments passed to this data setup script:
+
+**--contracts** : (Required) : The list of comma separated contracts to use for writing out the test urls. You can add any number of contracts here, and the data script will loop through a search of each contract and all of its pages and write them to file.
+
+**--year** : (Required) : The year to search by for contracts. This will be applied to all contracts from --contracts. This should be in YYYY format.
+
+**--month** : (Required) : The month to search by for contracts. This will be applied to all contracts from --contracts. This must be in MM format (i.e. 01 for January, 05 for May, etc)
+
+**--count** : (Required) : How many results per page to request. A lower number means you'll get more pages with smaller sized results, higher number means less pages with larger results. This can help test various sized payloads, or give tests more urls to use.
+
+**--version** : (Required) : The endpoint version to setup the data for. The only valid values for this (at time of writing) are "v1" or "v2". The resulting file will be prepended with this version so that multiple cursor files can be set up for different endpoints.
+
+The script will output a file named after the --version in the configured home directory, where the contract tests are expecting it. Make sure you run the script again if you need to set up data for a different endpoint version.
+>>>>>>> be38e90f64c4c2e8b2dd378e9e5e3bed5227f065
```

Comment:
Looks like there was a merge conflict here that slipped through -- I'm assuming that this section should be removed (excluding `--resetStats` documentation on line 114, obviously).

Reply:
And I fail at VS Code... Sorry about that!